Mapping/ClassMetadata.php

```diff
@@ -106,6 +115,22 @@ public function getIri()
     }
 
     /**
+     * @return string
+     */
+    public function getIdentifier()
+    {
+        return $this->identifier;
```

Comment:
Instead of having this new identifier property and the new `IdentifierLoader` and `DoctrineIdentifierLoader` classes, isn't it more simple to loop over attributes and use its `isIdentifier` method ? And it'd be the responsibility of attribute loader to look if it's the identifier, based on the different ways it wants (Doctrine annotation, `id` name, `Identifier` annotation, ...).